src/main/io/osd.c

```diff
@@ -435,6 +435,14 @@ static void osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_REM_TIM_EST:
+	if (!getMAhDrawn())
+		// FIXME add 's' to font
+		tfp_sprintf(buff, "NA S");
+	else
+		tfp_sprintf(buff, "%4dS", (int)((osdConfig()->cap_alarm - getMAhDrawn()) * ((float)flyTime / 1000000) / getMAhDrawn()));        
```

Comment:
I'd put `getMAhDrawn()` into a local variable - otherwise you might get different readings for the two calls if there ever is a current sensor that is updated in an ISR.

Also, I think splitting this into minutes:seconds will make it easier to absorb the information.

And thirdly, I am not sure what the negative value in the case of overrunning the capacity alarm is meant to convey. In my opinion, it should just be a blinking `0` in this case.